cmake/modules/SwiftComponents.cmake

```diff
@@ -122,3 +122,29 @@ function(swift_install_in_component component)
     install(${ARGN})
   endif()
 endfunction()
+
+function(swift_install_symlink_component component)
+  cmake_parse_arguments(
+      ARG # prefix
+      "" "LINK_NAME;TARGET;DESTINATION" "" ${ARGN})
```

Comment:
@llvm-beanz I disagree with you here. We are already in many parts of swift performing these annotations in the comments. It would be better to just make the code self document rather than rely on comments. *OR* we should strip out all of the comments. I would prefer that we be consistent here. Notice how you left in one of the comments (for the prefix), but removed the one for options, etc. We should be consistent here.